### DISCUSSION TITLE
Image resize tab

### DISCUSSION THREAD

**Discussion by web2299** [Uminosachi/sd-webui-inpaint-anything#134] [Ideas]
Can you  add image resize tab please ?
![image](https://github.com/Uminosachi/sd-webui-inpaint-anything/assets/159763476/a496b0bd-a7d1-49f1-a6bc-6555932f1b6f)

Thank you very much in advance for your work !

**Comment by Uminosachi** [comment]
Resizing images is complex, as it requires adjusting both the input and mask images to multiples of 8. Therefore, I would prefer not to add this feature at this time.

**Comment by web2299** [comment]
I see ok . It could be useful anyway ... Can you tell me the full path where are located the inpaint models like "realisticVisionV51_v51VAE-inpainting"  please ?

**Comment by Uminosachi** [comment]
The path is listed in the README at the following link section:

https://github.com/Uminosachi/sd-webui-inpaint-anything?tab=readme-ov-file#model-cache

**Reply by web2299** [reply]
ok thank you ! And one more question if you allow me - wonder if this model [GPEN-Inpainting-1024](https://public-vigen-video.oss-cn-shanghai.aliyuncs.com/robin/models/GPEN-Inpainting-1024.pth)  could be integrated in inpaint-anything ?  I think it could be useful . https://github.com/yangxy/GPEN
